lib/elixir/lib/code.ex

```diff
@@ -650,6 +650,10 @@ defmodule Code do
       when non-existing atoms are found by the tokenizer.
       Defaults to `false`.
 
+    * `:static_atom_encoder` - The static atom encoder function, see
+      "The `:static_atom_encoder` function" section below. This option
+      overrides the `:existing_atoms_only` option.
```

Comment:
I removed the link because it doesn't work nicely when read on the terminal. :(

Reply:
:(